File: coresuite-dotnet-standard-vb/merger/FormFlattening.vb
Imports ceTe.DynamicPDF.Forms
Imports ceTe.DynamicPDF.Merger

Namespace coresuite_dotnet_standard_vb.merger

    Public Class FormFlattening

        Public Shared Sub Run(outputPdfPath As String)

            Dim document As MergeDocument = New MergeDocument(Util.GetResourcePath("PDFs/fw9AcroForm_18_filled.pdf"))

            document.Form.Output = FormOutput.Flatten

            document.Draw(outputPdfPath)
        End Sub
    End Class
End Namespace

File: coresuite-dotnet-core-console-cs/merger/FormFlattening.cs
using ceTe.DynamicPDF.Forms;
using ceTe.DynamicPDF.Merger;

namespace coresuite_dotnet_core_console_cs.merger
{
    public class FormFlattening
    {
        public static void Run(string outputPdfPath)
        {
            MergeDocument document = new MergeDocument(Util.GetResourcePath("PDFs/fw9AcroForm_18_filled.pdf"));

            document.Form.Output = FormOutput.Flatten;

            document.Draw(outputPdfPath);
        }
    }
}